.github/pull_request_template.md

```diff
@@ -1,13 +1,25 @@
-<!--- Does this work have a corresponding ticket? --> 
+## Ticket
+<!---
+  Does this work have a corresponding ticket?
 
-https://smartcontract-it.atlassian.net/browse/...
+  Please link your Jira ticket by including it in one of the following reference:
+    - the PR title
+    - branch name
+    - commit message
+  
+  By referencing it, it will let the QA team to know what to watch out for when creating a new release.
+--> 
 
-<!--- Does this work depend on other open PRs? -->
+## Requires Dependencies
+<!---
+  Does this work depend on other open PRs?
 
-Requires:
-- https://github.com/smartcontractkit/chainlink-common/pull/123456
```

Comment:
I was trying to encourage a particular format here, because GitHub is picky about formatting these links, and a list item will show an icon indicating the status

Reply:
Can we put it inside this block comment?

Reply:
sure I'll update it with examples :)